lib/js_of_ocaml/import.ml

```diff
@@ -0,0 +1,71 @@
+(* Js_of_ocaml
```

Comment:
Is there a place we could put this Import module where it could be shared?

Reply:
I believe theses import files will diverge on purpose overtime.

Reply:
This diff has been pushed on master already.